packages/mds-metrics/utils.ts

```diff
@@ -15,12 +16,32 @@ export function getTimeBins({
   }))
 }
 
-export function getBinSizeFromQuery(query: MetricsApiRequest['query']) {
-  const bin_size_english: 'hour' | 'day' = query.bin_size || 'hour'
+export function convertBinSizeFromEnglishToMs(bin_size_english: HourOrDay) {
   const timeToMs = {
     hour: hours(1),
     day: days(1)
   }
   const bin_size = timeToMs[bin_size_english]
   return bin_size
 }
+
+export function normalizeToArray<T>(elementToNormalize: T | T[] | undefined): T[] {
+  let normalizedArray: T[]
+  if (elementToNormalize === undefined) {
+    normalizedArray = []
+  } else if (isArray(elementToNormalize)) {
+    normalizedArray = elementToNormalize
+  } else {
+    normalizedArray = [elementToNormalize]
+  }
+  return normalizedArray
+}
```

Comment:
Using the `let` here is unnecessary, could just add `return`s

Reply:
Also, if you add return statements, the linter will make all the else clauses go away.